modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceCompare.testcase

```diff
@@ -82,12 +82,43 @@ definition {
 	}
 
 	@description = "COMMERCE-5914. As a buyer, I want to deselect a item in Product Details for compare in Product Widget"
-	@ignore = "true"
 	@priority = "3"
 	test RemoveComparisonItemFromProductDetailsPage {
 		property portal.acceptance = "false";
 
-		// TODO COMMERCE-5914 RemoveComparisonItemFromProductDetailsPage pending implementation
+		CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+		ApplicationsMenu.gotoSite(site = "Minium");
+
+		Click.clickAtNotVisible(
+			key_productName = "U-Joint",
+			locator1 = "Checkbox#COMPARE_CHECKBOX_PRODUCT_CARD");
+
+		Click.clickAtNotVisible(
+			key_productName = "Transmission Fluid",
+			locator1 = "Checkbox#COMPARE_CHECKBOX_PRODUCT_CARD");
+
+		CommerceAcceleratorsInitializer.gotoMiniumProductDetailsPage(productName = "U-Joint");
+
+		AssertElementPresent(
+			key_productName = "u-joint",
+			locator1 = "CommerceAccelerators#CHECK_ITEM_COMPARE");
+
+		AssertElementPresent(
+			key_productName = "transmission-fluid",
+			locator1 = "CommerceAccelerators#CHECK_ITEM_COMPARE");
+
+		AssertChecked.assertCheckedNotVisible(
+			checkboxName = "Compare",
+			locator1 = "Checkbox#ANY_CHECKBOX_CHECKED");
+
+		Click.clickAtNotVisible(
+			checkboxName = "Compare",
+			locator1 = "Checkbox#ANY_CHECKBOX");
+
+		AssertElementNotPresent(
+			key_productName = "u-joint",
+			locator1 = "CommerceAccelerators#CHECK_ITEM_COMPARE");
```

Comment:
As for the comment above, please update `CommerceAccelerators#CHECK_ITEM_COMPARE` locator name.